src/frida/repl.py

```diff
@@ -325,6 +343,21 @@ def _process_message(self, message, data):
             else:
                 self._print("message:", message, "data:", data)
 
+        def _on_change(self, changed_file, other_file, event_type):
+            if changed_file != self._user_script:
+                return
```

Comment:
This check can be omitted if we only monitor the file and not its parent directory.